src/service/multisig.service.ts

```diff
@@ -0,0 +1,102 @@
+import {Address, AggregateTransaction} from 'nem2-sdk';
+import {Observable, Subject} from 'rxjs';
+import {startWith} from 'rxjs/operators';
+
+const SEQUENTIAL_FETCHER_DEFAULT_DELAY = 200;
+
+export class SequentialFetcher {
+ isFetching = false;
+ private routineController: Subject<boolean>;
+ private networkCallsIterator: AsyncGenerator;
+ private transactions$: Subject<AggregateTransaction[]>;
```

Comment:
I would avoid using "$" symbol.

Reply:
OK! I put it here to make it obvious it was an observable while keeping a shot name, but I agree it is a bit awkward since it would be the only variable to follow this convention